### ISSUE TITLE
Can't login to my Azure subscription when using storage explorer

### ISSUE THREAD

**Issue by nggpjoseph** [microsoft/AzureStorageExplorer#4410]
{
  "message": "\"{\\n  \\\"message\\\": \\\"\\\\\\\"The resource you are attempting to access is in a tenant which is currently filtered out. Un-filter it to access this resource.\\\\\\\"\\\"\\n}\""
}

**Comment by MRayermannMSFT** [comment]
@nggpjoseph can you try each of these:
- Go to the account management panel (person icon on the left), and check the checkbox for the tenant that the resource you are trying to access is in.
- If the checkbox is already checked, restart storage explorer, go to the account panel again, and make sure it is checked.
- If after restarting and re-checking the checkbox, the error still appears, then sign out and then back in, and then re-check the checkbox.

**Comment by MRayermannMSFT** [comment]
Closing due to lack of response. If you are still encountering this issue then please open a new issue here.
Thanks.

**Comment by WillStrohl** [comment]
@MRayermannMSFT This happened to me as well.  I opened the app and noticed a tenant was missing.  I looked at accounts, and noticed it said to re-authenticate.  I did that, but it didn't appear to do anything at first.  I think it was the third or fourth time when the authentication finally appeared in the app.  However, the tenant wouldn't show up again still.  I had to close and reopen the app to see it.
